system/Commands/Utilities/Routes.php

```diff
@@ -146,6 +146,11 @@ public function run(array $params)
             $tbody = [...$tbody, ...$autoRoutes];
         }
 
+        // Sort based on controller.
+        usort($tbody, function ($handler1, $handler2) {
+            return strcmp($handler1[2], $handler2[2]);
```

Comment:
Do you really need case-sensitive sorting? Why not use `strcasecmp`?

Reply:
Already fixed, thank u @paulbalandan